src/shogun/labels/MulticlassLabels.h

```diff
@@ -53,6 +53,12 @@ class CMulticlassLabels : public CDenseLabels
 
 		/** constructor
 		 *
+		 * @param src labels to set from a probability matrix
```

Comment:
You could say from a matrix that contains scores for each class for each point. Those could be probabilities but do not have to be normalized